source/class/cv/transforms/Mqtt.js

```diff
@@ -59,7 +59,7 @@ qx.Class.define('cv.transforms.Mqtt', {
               ret_pre = '',
               ret_post = '';
             // split on "." but not on "\." to allow the dot to be escaped
-            parameter.split(/(?<!\\)\./).forEach(
+            parameter.match(/(\\\.|[^.])+/g).forEach(
```

Comment:
I do not if it can happen in reality but this code would break if `parameter` is an empty string. So I would suggest to add a null check: `parameter.match(/(\\\.|[^.])+/g)?.forEach(`